# GraphMatcherMsg.msg
Header header
float32[] adjacency_matrix
int32[] indexed_matrix
float32[] isomorphism_list
float64[] 3D_coordinates
int32[] tags_id